CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Broad and vague policies that can be used to harass and exclude people from the project, including myself !
Similar code of conduct led to the exclusion of Linus Torvald from the Linux kernel project :
https://www.newyorker.com/science/elements/after-years-of-abusive-e-mails-the-creator-of-linux-steps-aside
We just don't need that.